src/data/data.js

```diff
@@ -139,4 +139,15 @@ const data = [
 	},
 ]
 
-export default data;
+function Portfolio({ id, name, link, tags }) {
+	this.id = id;
+	this.name = name;
+	this.link = link;
+	this.tags = tags;
+}
+
+Portfolio.prototype.belongsTo = function (tags = []) {
+	return tags.every(tag => this.tags.map(e => e.toLowerCase()).includes(tag));
+}
```

Comment:
What do you think about this ?
```suggestion
function Portfolio (portofolio = {}) {
        Object.assign(this, portofolio);
        this.belongsTo = (tags = []) => tags.every(tag => this.tags.map(e => e.toLowerCase()).includes(tag));
}
```